Gems/Multiplayer/Code/Source/Editor/MultiplayerEditorConnection.cpp

```diff
@@ -178,6 +178,18 @@ namespace Multiplayer
 
     void MultiplayerEditorConnection::OnDisconnect([[maybe_unused]] AzNetworking::IConnection* connection, [[maybe_unused]] DisconnectReason reason, [[maybe_unused]] TerminationEndpoint endpoint)
     {
-        ;
+        bool editorLaunch = false;
+        if (auto console = AZ::Interface<AZ::IConsole>::Get(); console)
```

Comment:
remove "; console"

Reply:
That would initialize console without checking the value.